.github/workflows/release-container.yml

```
@@ -232,3 +355,320 @@ jobs:
+            --arg cpu_image "$CPU_IMAGE" \
+            --arg cuda_image "$CUDA_IMAGE" \
+            '{llamarack:{version:$version,commit:$commit},llama_cpp:{release:$llama_release,build:$llama_build,cpu_runtime:$cpu_runtime,cuda_runtime:$cuda_runtime},images:{cpu:$cpu_image,cuda:$cuda_image}}' \
+            > artifacts/release/build-identity.json
+          gh release upload "$RELEASE_TAG" artifacts/release/build-identity.json --clobber
+
+          body="$(gh api "repos/${GITHUB_REPOSITORY}/releases/${RELEASE_ID}" --jq '.body // ""')"
+          marker='<!-- llamarack-runtime-identity -->'
+          if ! grep -Fq "$marker" <<< "$body"; then
+            section="$(cat <<EOF
+
+${marker}
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)